packages/keycodes/src/index.js

```diff
@@ -390,6 +390,14 @@ export const isKeyboardEvent = mapValues(
 		) => {
 			const mods = getModifiers( _isApple );
 			const eventMods = getEventModifiers( event );
+			/** @type {Record<string,string>} */
+			const replacementWithShiftKeyMap = {
+				Comma: ',',
+				Backslash: '\\',
+				// Windows returns `\` for both IntlRo and IntlYen.
```

Comment:
This comment was taken from [the source code of chromium](https://chromium.googlesource.com/chromium/src/+/lkgr/ui/events/keycodes/dom/dom_keyboard_layout_map_win.cc#110)